PeerioTechnologies/peerio-desktop
# fix: preserve line breaks in plain-text messages

#### Relevant info and issue/PR links 
 
https://app.clubhouse.io/peerio/story/9382/plaintext-messages-don-t-have-line-breaks

#### Testing instructions  

Send the following message:

```
Hello world
http://clipart-library.com/img1/909785.jpg
```

(line breaks may be mangled by copying, so type out "Hello world", press Shift+Enter and then paste the link)

Check that the image preview appears and there's no `ERROR Error: worldhttp://clipart-library.com/img1/909785.jpg request failed` in console.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
